src/track/playcounter.h

```diff
@@ -40,6 +39,7 @@ class PlayCounter {
 private:
     int m_iTimesPlayed;
     bool m_bPlayed;
+    std::shared_ptr<ControlProxy> m_pPracticemodeEnabled;
```

Comment:
The style in mixxx is just to use ControlProxy*.  Don't worry about freeing the pointer we handle that.

Reply:
How is that handeled? like when I look into that comment it's like hey take care of your things: https://github.com/mixxxdj/mixxx/pull/2431#discussion_r364600741

Reply:
oh oops.  hmm.

Reply:
I don't know what our policy is on using sharedpointer to avoid needing to add things to the destructor.  @uklotzde ? @daschuer ?